testsuite/integration/manualmode/src/test/resources/legacy-configs/host/eap-7-4-0.xml

```diff
@@ -49,8 +49,6 @@
             <heap size="64m" max-size="256m"/>
             <jvm-options>
                 <option value="-server"/>
-                <option value="-XX:MetaspaceSize=96m"/>
-                <option value="-XX:MaxMetaspaceSize=256m"/>
```

Comment:
Please restore these. This config is meant to match as much as possible what was shipped in EAP 7.4.

Reply:
Sure, I have restored it.